src/context/ThemeProvider.tsx

```diff
@@ -0,0 +1,24 @@
+import { ReactNode, useState } from "react";
+import { ThemeContext } from "./ThemeContext";
+
+type Props = {
```

Comment:
Props is not a good name for this, and use interface instead of type, somehting like
interface IThemeProviderProps {
...
}